src/ForkCMS/Bundle/InstallerBundle/Resources/public/js/backend.js

```diff
@@ -21,33 +21,32 @@ jsBackend.controls = {
 
   bindPasswordStrengthMeter: function () {
```

Comment:
wouldn't it make more sense to call this installer.js?

Reply:
maybe, butt i guess more than only this can be moved. Isn't it there for a reason?

Reply:
I think it is because it was originally copied from the backend